translated: docs/developer-docs/integrations/index.md

hokosugi opened the pull request:
### 修正依頼点について

* 41行目 意訳しています。。カッコ内は Neuronに掛かっていると思いますが、良い訳が見つからず、「ガバナンスに参加する観点から」と中途半端な終わり方になっています。

以上です。

尚、本家がSNSをアップした関係上、見逃して追いましたが、index.mdにもSNSの項目がありました。今回はこのまま訳を終えて、すぐに本家に追随する作業をして、SNSフォルダの翻訳にかかります。

宜しくおねがいします。

hokosugi committed:
translated: docs/developer-docs/integrations/index.md

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+これまでのセクションでは、IC 上で Canister を作り始める方法を案内してきましたが、ここでは、様々な（時には最先端の）追加機能を Dapps に統合する方法を紹介します。
+
+## Bitcoin 統合
+Bitcoin ネットワークと直接統合し、IC 上の Canister が Bitcoin を受け取り、保有し、送信することができ、すべてBitcoin ネットワーク上のトランザクションと直接統合します。つまり、Canister は、Bitcoin ネットワーク上で Bitcoin を保有する一般ユーザーと全く同じように行動することができます。
```
直接統合が２回出てくるので少し修正しました
```suggestion
Bitcoin ネットワークと直接統合することで、IC 上の Canister が Bitcoin を受け取り、保有し、送信することを、すべてBitcoin ネットワーク上のトランザクションで直接行うことができます。つまり、Canister は、Bitcoin ネットワーク上で Bitcoin を保有する一般ユーザーと全く同じように行動することができます。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+* [Bitcoin Dapps をローカルで開発する](./bitcoin/local-development.md)ではローカル環境での開発の仕方を説明しています。
+
+## 閾値 ECDSA
+ブロックチェーン上の閾値 ECDSA 実装は、秘密鍵を安全に保管し、適格なエンティティの要求に応じて、それらにのみ署名を発行するハードウェアセキュリティモジュール（HSM）に対するオンチェーン版と見なすことができます。特に（ECDSA ベースの）ブロックチェーンとの直接統合を促進する際に重要になります。
```
nitsです
```suggestion
ブロックチェーン上の閾値 ECDSA 実装は、秘密鍵を安全に保管し、適格なエンティティの要求に応じて、それらにのみ署名を発行することができ、これはハードウェアセキュリティモジュール（HSM）のオンチェーン版と見なすことができます。特に（ECDSA ベースの）ブロックチェーンとの直接統合を促進する際に重要になります。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+* [技術概要 - 仕組み](./t-ecdsa/t-ecdsa-how-it-works.md)ではより詳細に説明しています。
+
+## HTTPS アウトコール
+IC 上の HTTP(S) アウトコールにより、 Canister はブロックチェーンの外部の HTTP(S) サーバーに直接コールし、処理をした上でそのレスポンスを Canister で使用し、それらの入力された（コンセンサス済みの）複製されたステートを安全に更新できるようになります。ブロックチェーン史上初の試みであり、オラクルの必要性を軽減するものです。
```
using those inputsなので「入力を使って」の方が良いと思います
```suggestion
IC 上の HTTP(S) アウトコールにより、 Canister はブロックチェーンの外部の HTTP(S) サーバーに直接コールし、処理をした上でそのレスポンスを Canister で使用し、この入力を使って（コンセンサス済みの）複製されたステートを安全に更新できるようになります。ブロックチェーン史上初の試みであり、オラクルの必要性を軽減するものです。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+
+## Internet Identity
+Internet Identity は、Web3 サービスや Dapps とのセッションを作成したり、従来のブロックチェーン取引に署名したりすることができます。
+* [Internet Identity](./internet-identity/integrate-identity.md) は、アプリで Internet Identity を使用方法の概要を説明します。
```
「リンク先のページでは、○○を説明している」という意味となるように、「〜〜では、」とするのはどうでしょうか。（「〜〜は」、とするとリンク先という意味が読み取りづらいように感じました）
以下、同じ修正観点でコメントなしでsuggestionsつけている箇所あるのでご確認ください。
```suggestion
* [Internet Identity](./internet-identity/integrate-identity.md) では、アプリで Internet Identity を使用する方法の概要を説明します。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+## ICP 台帳
+Internet Computer プロトコル（ICP）は、台帳 Canister と呼ばれる特殊な Canister を使用して ICP の管理を実装しています。台帳 Canister はひとつで、NNS のサブネット上で他の Canister と一緒に稼働します。台帳 Canister は、アカウントとトランザクションを保持するスマートコントラクトです。 
+
+* [ICP 台帳](./ledger/index.md)は ICP 台帳の基礎を説明しています。
```
```suggestion
* [ICP 台帳](./ledger/index.md)では、 ICP 台帳の基礎を説明しています。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+
+* [ICP 台帳](./ledger/index.md)は ICP 台帳の基礎を説明しています。
+* [ICP 台帳との連携](./ledger/interact-with-ledger.md)では、ICP 台帳を操作するためのコマンドとプロトコルフローを示します。
+* [台帳のローカルセットアップ](./ledger/ledger-local-setup.md)は、ローカル環境で台帳の利用方法を説明しています。
```
```suggestion
* [台帳のローカルセットアップ](./ledger/ledger-local-setup.md)では、ローカル環境での台帳の利用方法を説明しています。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+* [ICP 台帳](./ledger/index.md)は ICP 台帳の基礎を説明しています。
+* [ICP 台帳との連携](./ledger/interact-with-ledger.md)では、ICP 台帳を操作するためのコマンドとプロトコルフローを示します。
+* [台帳のローカルセットアップ](./ledger/ledger-local-setup.md)は、ローカル環境で台帳の利用方法を説明しています。
+* [新しいトークンのデプロイ](./ledger/deploy-new-token.md)は、新しいトークンをデプロイして、Rosetta に接続する方法を説明しています。
```
```suggestion
* [新しいトークンのデプロイ](./ledger/deploy-new-token.md)では、新しいトークンをデプロイして、Rosetta に接続する方法を説明しています。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+## Rosetta
+Rosetta は、Coinbase が導入したオープンスタンダードで、取引所、ブロックエクスプローラー、ウォレットにおけるブロックチェーンベースのトークン統合を簡素化するためのものです。このドキュメントは、CeFi 取引所で取引可能なことを目的としたトークンを IC 上にデプロイしたい場合や、ブロックエクスプローラーやウォレットに取り組んでいる場合に役立つはずです。
+
+* [Rosetta](./rosetta/index.md) は、Rosetta ノードの設定方法とよくある質問に対する回答をしています。
```
```suggestion
* [Rosetta](./rosetta/index.md) では、Rosetta ノードの設定方法とよくある質問に対する回答をしています。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+Rosetta は、Coinbase が導入したオープンスタンダードで、取引所、ブロックエクスプローラー、ウォレットにおけるブロックチェーンベースのトークン統合を簡素化するためのものです。このドキュメントは、CeFi 取引所で取引可能なことを目的としたトークンを IC 上にデプロイしたい場合や、ブロックエクスプローラーやウォレットに取り組んでいる場合に役立つはずです。
+
+* [Rosetta](./rosetta/index.md) は、Rosetta ノードの設定方法とよくある質問に対する回答をしています。
+* [スタンダードトークンを送金する](./rosetta/transfers.md)は、Rosetta Construction API を使用した ICP の転送方法について説明しています。
```
```suggestion
* [スタンダードトークンを送金する](./rosetta/transfers.md)では、Rosetta Construction API を使用した ICP の転送方法について説明しています。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+
+* [Rosetta](./rosetta/index.md) は、Rosetta ノードの設定方法とよくある質問に対する回答をしています。
+* [スタンダードトークンを送金する](./rosetta/transfers.md)は、Rosetta Construction API を使用した ICP の転送方法について説明しています。
+* [Neuron のライフサイクル](./rosetta/neuron-lifecycle.md)は、Neuron の概要を説明します（IC アセットにより、コントローラーがプロポーザルの提出や投票によってネットワークのガバナンスに参加する観点から）。
```
以下はどうでしょうか
```suggestion
* [Neuron のライフサイクル](./rosetta/neuron-lifecycle.md)では、Neuron の概要を説明します（Neuronとは、コントローラーがプロポーザルの提出や投票によってネットワークのガバナンスに参加することを可能とする IC のアセットです）。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+* [Rosetta](./rosetta/index.md) は、Rosetta ノードの設定方法とよくある質問に対する回答をしています。
+* [スタンダードトークンを送金する](./rosetta/transfers.md)は、Rosetta Construction API を使用した ICP の転送方法について説明しています。
+* [Neuron のライフサイクル](./rosetta/neuron-lifecycle.md)は、Neuron の概要を説明します（IC アセットにより、コントローラーがプロポーザルの提出や投票によってネットワークのガバナンスに参加する観点から）。
+* [ステーキングと Neuron 管理](./rosetta/staking-support.md)は、Rosetta API を拡張し、IC 上で資金のステーキングやガバナンス Neuron の管理を可能にすることを説明しています。
```
```suggestion
* [ステーキングと Neuron 管理](./rosetta/staking-support.md)では、Rosetta API を拡張し、IC 上で資金のステーキングやガバナンス Neuron の管理を可能にすることについて説明しています。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+* [スタンダードトークンを送金する](./rosetta/transfers.md)は、Rosetta Construction API を使用した ICP の転送方法について説明しています。
+* [Neuron のライフサイクル](./rosetta/neuron-lifecycle.md)は、Neuron の概要を説明します（IC アセットにより、コントローラーがプロポーザルの提出や投票によってネットワークのガバナンスに参加する観点から）。
+* [ステーキングと Neuron 管理](./rosetta/staking-support.md)は、Rosetta API を拡張し、IC 上で資金のステーキングやガバナンス Neuron の管理を可能にすることを説明しています。
+* [Rosetta ステーキングのチュートリアル](./rosetta/staking-tutorial.md)は、Neuron を作成するプロセスを説明しています。
```
```suggestion
* [Rosetta ステーキングのチュートリアル](./rosetta/staking-tutorial.md)では、Neuron を作成するプロセスを説明しています。
```

hoosan commented on docs/developer-docs/integrations/index.md:
```diff
@@ -1,3 +1,49 @@
+* [Neuron のライフサイクル](./rosetta/neuron-lifecycle.md)は、Neuron の概要を説明します（IC アセットにより、コントローラーがプロポーザルの提出や投票によってネットワークのガバナンスに参加する観点から）。
+* [ステーキングと Neuron 管理](./rosetta/staking-support.md)は、Rosetta API を拡張し、IC 上で資金のステーキングやガバナンス Neuron の管理を可能にすることを説明しています。
+* [Rosetta ステーキングのチュートリアル](./rosetta/staking-tutorial.md)は、Neuron を作成するプロセスを説明しています。
+* [ホットキー](./rosetta/hotkeys.md)は、Neuron 管理用ホットキーの生成方法について説明しています。
```
```suggestion
* [ホットキー](./rosetta/hotkeys.md)では、Neuron 管理用ホットキーの生成方法について説明しています。
```

hoosan reviewed (changes requested):
翻訳ありがとうございます。
ご依頼の点含めコメントつけました。

本家の方でSNSの項目出ている件承知しました。Issueの方も修正しておきます。
本家追従作業お手数ですがよろしくお願いします🙏

hokosugi committed:
Apply suggestions from code review

レビューありがとうございました。「Neuronとは、コントローラーがプロポーザルの提出や投票によってネットワークのガバナンスに参加することを可能とする IC のアセットです」の訳はさすがですｗ

Co-authored-by: hoosan <40290137+hoosan@users.noreply.github.com>
